runtime/queries/metricsview_comparison_toplist.go

```diff
@@ -408,16 +414,32 @@ func (q *MetricsViewComparisonToplist) buildMetricsComparisonTopListSQL(mv *runt
 		q.Limit = 100
 	}
 
+	baseLimitClause := ""
+	comparisonLimitClause := ""
+
+	joinType := "FULL"
+	if q.Approximate {
+		if q.Sort[0].Type == runtimev1.MetricsViewComparisonSortType_METRICS_VIEW_COMPARISON_SORT_TYPE_BASE_VALUE ||
+			q.Sort[0].Type == runtimev1.MetricsViewComparisonSortType_METRICS_VIEW_COMPARISON_SORT_TYPE_ABS_DELTA ||
+			q.Sort[0].Type == runtimev1.MetricsViewComparisonSortType_METRICS_VIEW_COMPARISON_SORT_TYPE_REL_DELTA {
+			joinType = "LEFT OUTER"
+			baseLimitClause = fmt.Sprintf("LIMIT %d", q.Limit)
+		} else if q.Sort[0].Type == runtimev1.MetricsViewComparisonSortType_METRICS_VIEW_COMPARISON_SORT_TYPE_COMPARISON_VALUE {
+			joinType = "RIGHT OUTER"
+			comparisonLimitClause = fmt.Sprintf("LIMIT %d", q.Limit)
+		}
```

Comment:
What about using a greater limit for the approximation than `q.Limit` – e.g. if `q.Limit` is 10, we maybe would still do the approximate comparison against the top 200 values or something. What does the legacy dash do?